app/code/Magento/Sales/Model/Order.php

```diff
@@ -2112,6 +2112,7 @@ public function getAdjustmentPositive()
 
     /**
      * Returns applied_rule_ids
+     * Rules are comma separated if there are more than one.
```

Comment:
Please include a blank line before or add the "(comma separated)" information in return line, removing this one.